packages/html/src/editor/tests/editor-table-wizard.tsx

```diff
@@ -0,0 +1,280 @@
+            <section>
+                <Window className="k-editor-window k-editor-table-wizard-window" title="Table properties" actions={[ 'minimize', 'maximize', 'close' ]} actionButtonsAlign="start" actionButtons={
+                    <>
+                        <Button themeColor="primary">Save</Button>
+                        <Button>Cancel</Button>
+                    </>
+                }>
+                    <div className="k-tabstrip k-tabstrip-top">
+                        <div className="k-tabstrip-items-wrapper k-hstack">
+                            <ul className="k-tabstrip-items k-reset">
+                                <li className="k-item k-first k-active"><span className="k-link">General</span></li>
+                                <li className="k-item k-last"><span className="k-link">Advanced</span></li>
+                            </ul>
+                        </div>
+                        <div className="k-tabstrip-content k-active" >
+                            <Form tag="div" layout="grid" cols={4} gapX={4}>
+                                <FormField
+                                    className="k-col-span-2"
+                                    label="Rows"
+                                    editor={ < NumericTextbox value="3" /> }
+                                />
+                                <FormField
+                                    className="k-col-span-2"
+                                    label="Columns"
+                                    editor={ < NumericTextbox value="4" /> }
+                                />
+                                <FormField
+                                    className="k-col-span-1"
+                                    label="Width"
+                                    editor={ < NumericTextbox /> }
+                                />
+                                <FormField
+                                    className="k-col-span-1"
+                                    label="&nbsp;"
+                                    editor={ < DropdownList placeholder="%" /> }
+                                />
+                                <FormField
+                                    className="k-col-span-1"
+                                    label="Height"
+                                    editor={ < NumericTextbox placeholder="Auto" /> }
+                                />
+                                <FormField
+                                    className="k-col-span-1"
+                                    label="&nbsp;"
+                                    editor={ < DropdownList placeholder="px" /> }
+                                />
+                                <FormField
+                                    className="k-col-span-2"
+                                    label="Position"
+                                    editor={ < DropdownList prefix={ < Icon name="table-position-left" /> } placeholder="Left" />}
```

Comment:
Remove space between `< Icon`

Use the _correct_ names for all icons from: https://github.com/telerik/kendo-icons/blob/develop/src/telerik-icons/icons.json

table-position-left -> table-position-start